ChangeLog.d/mpi_read_negative_zero.txt

```diff
@@ -0,0 +1,3 @@
+Bugfix
+   * Fix mbedtls_mpi_read_string on "-0" returning a ``negative zero'' object,
+     which the library does fully consistently treat as equal to zero.
```

Comment:
I find this difficult to read and understand what the behaviour was before and what the behaviour is now.

Suggestion:
```suggestion
     which the library now consistently treats as equal to zero.
```